src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,236 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+import duke.exception.DukeException;
+import duke.ui.Ui;
+
+/**
+ * Initialises a list to store all tasks captured in duke.
+ */
+public class TaskList {
+
+    /** A list to store all user tasks. */
+    private ArrayList<Task> listOfTasks;
+    private ArrayList<String> preProcessedTasks;

```

Comment:
I also liked how you use the appropriate singular and plural nouns in naming your variables :D